tests/core/focusmanager/lostfocus.js

```diff
@@ -0,0 +1,29 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: divarea */
+
+bender.editor = {
+	extraPlugins: 'divarea',
+	allowedContent: true
+};
+
+bender.test({
+	'test lost focus in editor after click': function() {
+		// #17020 - test for checking if focusmanager reset ranges in divarea editor (bug introduced in #13446).
```

Comment:
Since https://github.com/ckeditor/ckeditor-dev/pull/470 got in, please update the reference to Trac issue into a full URL.